src/guard.rs

```diff
@@ -104,6 +104,10 @@ pub fn Any<F: Guard + 'static>(guard: F) -> AnyGuard {
 pub struct AnyGuard(Vec<Box<dyn Guard>>);
 
 impl AnyGuard {
+    /// Create AnyGuard from a vector of Guards
+    pub fn new(guards: Vec<Box<dyn Guard>>) -> Self {
+        AnyGuard(guards)
+    }
```

Comment:
Seems to me this change is unnecessary, it should be possible to use `Any(first_guard).or(second_guard)...`.

Reply:
I think it is necessary since there is no way (_at least for me_) to repeatedly call `Any(first_guard).or(second_guard)` when doing code generation using `quote!` in `actix-web-codegen`